### PR TITLE
Fixing some orthographical mistakes in PSR-1 and PSR-2 papers.

### PR DESCRIPTION
**Pull request by IlyaBakhlin**
Some changes I would like to apply to this document.

### CODE REVIEW

**Comment by IlyaBakhlin** [comment]
I've made some changen in the PSR-2 file.

File: accepted/PSR-2-coding-style-guide.md [hunk 1]
```diff
@@ -366,13 +366,13 @@ $foo->bar(
 
 The general style rules for control structures are as follows:
 
-- There MUST be one space after the control structure keyword
-- There MUST NOT be a space after the opening parenthesis
-- There MUST NOT be a space before the closing parenthesis
+- There MUST be one space after the control structure keyword.
```
Comment (GrahamCampbell) [inline accepted/PSR-2-coding-style-guide.md]: all these fullstops are wrong

File: accepted/PSR-2-coding-style-guide.md [hunk 2]
```diff
@@ -604,19 +604,19 @@ $foo->bar(
 There are many elements of style and practice intentionally omitted by this
 guide. These include but are not limited to:
 
-- Declaration of global variables and global constants
+- Declaration of global variables and global constants.
```
Comment (GrahamCampbell) [inline accepted/PSR-2-coding-style-guide.md]: no

File: accepted/PSR-2-coding-style-guide.md [hunk 3]
```diff
@@ -604,19 +604,19 @@ $foo->bar(
...
 
-- Declaration of functions
+- Declaration of functions.
```
Comment (GrahamCampbell) [inline accepted/PSR-2-coding-style-guide.md]: no

File: accepted/PSR-2-coding-style-guide.md [hunk 4]
```diff
@@ -604,19 +604,19 @@ $foo->bar(
...
 
-- Operators and assignment
+- Operators and assignment.
```
Comment (GrahamCampbell) [inline accepted/PSR-2-coding-style-guide.md]: etc...

File: accepted/PSR-1-basic-coding-standard.md [hunk 5]
```diff
@@ -17,19 +17,19 @@ interpreted as described in [RFC 2119].
 1. Overview
 -----------
 
-- Files MUST use only `<?php` and `<?=` tags.
+- Files MUST use only `<?php ?>` and `<?= ?>` tags.
```
Comment (samdark) [inline accepted/PSR-1-basic-coding-standard.md]: It's OK as is/

File: accepted/PSR-1-basic-coding-standard.md [hunk 6]
```diff
@@ -17,19 +17,19 @@ interpreted as described in [RFC 2119].
...
 
 - Files MUST use only UTF-8 without BOM for PHP code.
 
 - Files SHOULD *either* declare symbols (classes, functions, constants, etc.)
-  *or* cause side-effects (e.g. generate output, change .ini settings, etc.)
+  *or* cause side-effects (e.g. generate output, change INI settings, etc.)
```
Comment (samdark) [inline accepted/PSR-1-basic-coding-standard.md]: OK as is.

File: accepted/PSR-1-basic-coding-standard.md [hunk 7]
```diff
@@ -57,12 +57,12 @@ declaring classes, functions, constants, etc., *merely from including the
 file*.
 
 "Side effects" include but are not limited to: generating output, explicit
-use of `require` or `include`, connecting to external services, modifying ini
+use of `require` or `include`, connecting to external services, modifying INI
```
Comment (samdark) [inline accepted/PSR-1-basic-coding-standard.md]: OK as is.

**Comment by IlyaBakhlin** [comment]
Fixed. :v: :sweat_smile: 
Let me know if I missed something or if there's still something wrong.

**Comment by michaelcullum** [comment]
I'm not really sure I see a huge amount of benefit to these changes and as the spec is finalised, unless there is a good reason to change it, I'd rather leave it as-is.

**Comment by IlyaBakhlin** [comment]
Okay. :+1:

**Pull request closed without merging**